tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -353,7 +355,7 @@ def calibrate(self,
           fetches, feed_dict=feed_dict_fn() if feed_dict_fn else None)
 
     self.finalize_calibration()
-    return self._converted_graph_def
+    return self._converted_graph_def, self._calibration_tables
```

Comment:
Please add an attribute to TrtGraphConverter to access the calibration table instead.